command/rootchain/validators/validator_info.go

```diff
@@ -95,7 +95,7 @@ func runCommand(cmd *cobra.Command, _ []string) error {
 	stakeManagerAddr := types.StringToAddress(params.stakeManagerAddress)
 
 	validatorInfo, err := rootHelper.GetValidatorInfo(validatorAddr,
-		supernetManagerAddr, stakeManagerAddr, params.chainID, txRelayer)
```

Comment:
Move it to the `sidechain` commands, and modify it to call `StakeManager` on child chain. This means that we won't need `supernet-manager` nor `stake-manager` flags, since we know that `StakeManager` was deployed on genesis on a hardcoded address.